src/components/card/CreateNoteCard.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from 'react'
+import ReactQuill from 'react-quill';
+
+import './CreateNoteCard.css';
+import 'react-quill/dist/quill.snow.css';
+import './quill.css';
+
+import LabelCard from './LabelCard';
+import { useNotes } from '../../context/NotesContext';
+import ColorPalette from './ColorPalette';
+
+
+
+function CreateNoteCard() {
+
+    const { note,bgColor } =useNotes();
+
+  return <>
+  
+    <div className='create-note' >
+
+                  <div className={`create-note-container ${bgColor}`}>
+                     
+                        <div> 
+                            <input className={`note-title ${bgColor} `} type="text" placeholder="  Take notes....." />
+                        </div>
+
+                        <div className='app'>
+                            <ReactQuill bounds={'.app'}  /> 
+                        </div>
+                        <div className='note-tag-container' >
+
+                            {note.tags.map((tag,index)=><div key={index} className='note-tag' >{tag}</div> ) }
```

Comment:
Suggestion: try to destructure beforehand for long chaining eg. you could've destructured tags from note beforehand.

Reply:
ohk , i will do that beforehand